src/components/LiquidityMining/Create/Steps/index.tsx

```diff
@@ -14,13 +14,14 @@ const Root = styled(AutoColumn)<{ disabled: boolean }>`
   pointer-events: ${({ disabled }) => (disabled ? 'none' : 'auto')};
   transition: opacity 0.3s ease;
   opacity: ${({ disabled }) => (disabled ? 0.3 : 1)};
-  z-index: 1;
+  z-index: auto;
+  text-transform: capitalize;
 `
 
 export default function Step({ disabled, index, title, children, ...rest }: StepProps) {
   return (
     <Root gap="31px" disabled={disabled} {...rest}>
-      <TYPE.subHeader fontSize={18} lineHeight="22px" color="text4">
+      <TYPE.subHeader fontSize={18} lineHeight="22px" color="#EBE9F8">
```

Comment:
This color is in the theme but is defined as `bg`, I think we should use it with the theme but define it with a generic name.